docs/area-owners.md

```diff
@@ -16,13 +16,13 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-AssemblyLoader-mono                       | @agocke              | @agocke @elinor-fung                                |                                                                                                                                                                                                                                                                                           |
 | area-Build-mono                                | @lewing              | @akoeplinger                                        |                                                                                                                                                                                                                                                                                           |
 | area-Codeflow                                  | @dotnet/dnr-codeflow | @dotnet/dnr-codeflow                                | Used for automated PRs that ingest code from other repos                                                                                                                                                                                                                                  |
-| area-Codegen-AOT-mono                          | @steveisok           | @vargaz @kotlarmilos                                |                                                                                                                                                                                                                                                                                           |
+| area-Codegen-AOT-mono                          | @steveisok           | @kotlarmilos                                |                                                                                                                                                                                                                                                                                           |
```

Comment:
Could you please fix the table alignment so that it remains readable as plain text?